packages/fuels/Cargo.toml

```diff
@@ -32,10 +32,12 @@ fuels-test-helpers = { workspace = true }
 hex = { workspace = true, default-features = false }
 sha2 = { workspace = true }
 tempfile = { workspace = true }
-tokio = { workspace = true }
+tokio = { workspace = true, features = ["time", "test-util"] }
 
 [features]
 default = ["std", "fuels-test-helpers?/fuels-accounts"]
+coin-cache = ["fuels-accounts/coin-cache"]
```

Comment:
should this be used by default ?

Reply:
@digorithm

Reply:
I think so, yes. My rationale is that this feature is mostly to protect nodes against heavy querying. I don't see application builders being too mindful of this, so having this on by default should be safer. However, these users might run into caching-related issues, and then they will be mindful of checking this feature and, if needed, disabling it.